.github/workflows/ci-release.yaml

```diff
@@ -107,7 +107,7 @@ jobs:
           GH_TOKEN: ${{ steps.commit-token.outputs.token }}
         with:
           files: actions/**/action.yaml
-          commit-message: Update action.yaml files to ${{ env.image-name }}:${{ steps.semver.outputs.next }}
+          commit-message: 'chore: point Docker container actions to ${{ env.image-name }}:${{ steps.semver.outputs.next }}'
```

Comment:
```suggestion
          commit-message: 'chore(automation): point Docker container actions to ${{ env.image-name }}:${{ steps.semver.outputs.next }}'
```

To at least add some distinction between man-made commits and automated commits? Purely a suggestion

Reply:
I don't think that's a proper usage of the scope section of a conventional commit:
> A scope MUST consist of a noun describing a section of the codebase surrounded by parenthesis, e.g., `fix(parser):`

https://www.conventionalcommits.org/en/v1.0.0/#specification

Additionally, the commit author is IMHO more than enough to distinguish:
<img width="533" alt="image" src="https://github.com/user-attachments/assets/417309ba-24f2-43bd-ba02-e1fc7719824b">